lib/perimeterx/internal/perimeter_x_context.rb

```diff
@@ -58,9 +58,16 @@ def initialize(px_config, req)
           end
       end
       @context[:http_method] = req.method
-
+	  @context[:sensitive_route] = check_sensitive_route(px_config[:sensitive_routes], @context[:uri]) 
     end #end init
 
+	def check_sensitive_route(sensitive_routes, uri)
```

Comment:
px_config is not part of the instance itself so I would need to pass it, for the URI its not really mater if I pass it or not, It is more flexible to test in the feature